README.md

```diff
@@ -368,7 +368,9 @@ This is how we create and publish a new release of the 121-platform.
 - Check the changes since the last release, by replacing `vX.X-X` with the latest release in this URL: `https://github.com/global-121/121-platform/compare/vX.X-X...main`  
   Check any changes to:
   - `services/.env.example`:  
-     If there are, then make any configuration changes to the staging-service(_Or Mock-Service_) in the Azure Portal, relating to new/changed/removed `ENV`-variables, changed default values, etc.
+     If there are:
+    - related to the 121-service: Use the GitHub Actions-workflow: [`infrastructure`: Set Azure WebApp Environment variable](https://github.com/global-121/infrastructure/actions/workflows/set_azurewebapp_env-variable.yml) (Set the instance to `staging`!)
```

Comment:
how do we feel about the code repo referencing the infrastructure repo? Should it be stand-alone an infrastructure-agnostic in principal? Although I guess we're already not fully living by this guideline..

Reply:
Good to know there is that item on the backlog. I would try to close off this item and then suggest to priotize that new one. Remains for now the question if we're OK with the current changes (knowing we will solve it later), or if we already "don't want to make it worse at least". I don't know what the simpler alternative in that latter case would be though. We can discuss if you want.

Reply:
We prioritized AB#29080 a bunch so it's on our radar more.